app/models/custom_hint.rb

```diff
@@ -64,7 +65,13 @@ def validate_csv
   def process_records
     validate_csv
     CSV.parse(@csv, headers: true) do |record|
-      if %w(Title URL Fingerprint).none? { |header| record[header].nil? }
+      if %w(Title URL).all? { |header| record[header].present? }
+        # Add the fingerprint if needed; people creating spreadsheet rows may
```

Comment:
Does there remain value in storing the fingerprint for _some_ records in the spreadsheet? Or should we always just ensure there is an example search and generate a fingerprint from that?